communitter-frontend/src/utilities/ProfileService.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import {tokenName, url} from "./config";
+
+export class ProfileService {
+
+
+    static async uploadProfilePicture(image, userId) {
+
+        const token = localStorage.getItem(tokenName);
+
+        if (!token) {
+            throw new Error("Authentication token not found");
+        }
+
+        const formData = new FormData();
+        formData.append("image", image);
+
+        try {
+            const response = await axios.post(
+                `${url}/Images/user/${userId}/profile-picture`,
```

Comment:
Images should be lowercase for the endpoint

Reply:
Handled